addons/beehave/debug/graph_node.gd

```diff
@@ -0,0 +1,133 @@
+@tool
+extends GraphNode
+
+const DEFAULT_COLOR := Color("#dad4cb")
```

Comment:
Not for this PR but something to think about in future: do we want to specify the "beehave colours" in some sort of theme resource and ship it with the addon? These colours could then be accessed from that theme directly.